src/scss/components/_fullscreen-pop-over.scss

```diff
@@ -37,16 +37,19 @@ $fullScreenPopOver-shadow-color      : rgba($black, 0.12);
 }
 
     // This style block is for accessibility - it's so the user can't get caught tabbing
-    // within a modal if it's not visible
-    .c-fullScreenPopOver {
+    // within a modal if it's not visible on mid and below
+    .c-fullScreenPopOver--midNoTab {
```

Comment:
The convention I've seen used elsewhere, such as the utility classes, is something like `c-fullScreenPopOver--noTab--mid`.